importer/tests/test_views.py

```diff
@@ -20,17 +23,113 @@
 class TestViews(LoreTestCase):
     """Hit each view."""
 
+    import_url = "/repositories/{0}/import/"
+    repository_url = "/repositories/{0}/"
```

Comment:
Actually, I take that back. @giocalitri, I think this should definitely be in a `setUp` override, mostly because then `self.repo` is in scope, and the URL can have `self.repo.id` only added once, rather than passed in many times  during the test.